src/shared_modules/utils/threadEventDispatcher.hpp

```diff
@@ -165,6 +165,16 @@ class TThreadEventDispatcher
         }
     }
 
+    uint64_t bulkSize() const
+    {
+        return m_bulkSize;
+    }
+
+    void bulkSize(const uint64_t bulkSize)
+    {
+        m_bulkSize = bulkSize;
```

Comment:
Question: is it safe to modify this value without a mutex/atomic variable?

Reply:
Good catch